code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -393,5 +393,6 @@
 	name = "organ freezer"
 	icon_state = "freezer_red"
 	item_state = "medicalpack"
-	matter = list(MATERIAL_PLASTEEL = 1, MATERIAL_PLASTIC = 2)
+	matter = list(MATERIAL_STEEL = 5, MATERIAL_PLASTIC = 15)
 	max_storage_space = DEFAULT_NORMAL_STORAGE * 1.25
```

Comment:
If you are limiting it to just organs, maybe buff the space as every organ is a normal size item iirc.

Reply:
That's what the 1.25 is meant for, it is already bigger than the regular freezer

I could increase it further if @SirRichardFrancis thinks it is a good idea